client/sites/marketing/tools/style.scss

```diff
@@ -20,6 +20,65 @@
 	background-color: #fff;
 }
 
+.search-component.search-categories__searchbox {
+	max-width: 250px;
+	width: 100%;
+	height: 40px;
+	z-index: 0;
+	margin-top: 24px;
+	margin-bottom: 16px;
+	
+	border-radius: 4px;
+
+	&.search-categories__searchbox--mobile {
+		max-width: none;
+	}
+
+	box-shadow: 0 0 0 1px var(--studio-gray-10);
+
+	.search-component__icon-navigation:focus {
+		box-shadow: none;
+	}
+	.search-component__open-icon,
+	.search-component__close-icon {
+		color: var(--studio-gray-30);
+
+		&:hover {
+			color: var(--studio-gray-60);
+		}
+	}
+
+	.search-component__icon-navigation {
+		margin-right: 4px;
+	}
+
+	.search-component__icon-navigation-separator {
+		user-select: none;
+		color: var(--studio-gray-5);
+		margin-bottom: 3px;
+	}
+
+	.search-component__input-fade {
+		font-size: $font-body-small !important;
+		padding-left: 14px !important;
+	}
+
+	.search-component__open-icon {
+		height: 24px;
+		margin-right: -9px;
+	}
+
+	.search-component__close-icon {
+		height: 20px;
+		margin-left: -9px;
+		margin-right: -10px;
+	}
+}
```

Comment:
Are the overrides related to the specific designs for the Marketing Tools? I believe most of them are related to search box width and height. Have you tried using `fitsContainer` attribute?

@JanaMW27 do we have a collection of different search boxes? I would like to avoid styling new ones each time to apply different colors and font-sizes.

Reply:
>Are the overrides related to the specific designs for the Marketing Tools? 

Yes

> Have you tried using `fitsContainer` attribute?

Not really, do you think it would solve the margins and padding there? Shall I give it a try?

Reply:
I have removed all overrides and the result  (in my opinion) is totally fine! I would leave the current option without overrides. Please let me know if you see any issues.